docs/apm/troubleshooting.asciidoc

```diff
@@ -47,37 +47,14 @@ This section can help with any of the following:
 There are a number of factors that could be at play here.
 One important thing to double-check first is your index template.
 
-*Index template*
-An APM index template must exist for the APM app to work correctly.
-By default, this index template is created by APM Server on startup.
-However, this only happens if `setup.template.enabled` is `true` in `apm-server.yml`.
-You can create the index template manually by running `apm-server setup`.
+*Index templates*
+For the APM app to work correctly, an index template must exist for each APM data stream.
+By default, {fleet} sets up APM index templates when the APM integration is installed.
 Take note that index templates *cannot* be applied retroactively -- they are only applied at index creation time.
 More information is available in {apm-guide-ref}/apm-server-configuration.html[Set up and configure].
 
-You can check for the existence of an APM index template using the
-{ref}/indices-get-template.html[Get index template API].
-If you're using the default index naming pattern, that request would be:
-
-[source,js]
---------------------------------------------------
-GET /_template/apm-{version}
---------------------------------------------------
-// CONSOLE
-
-*Using Logstash, Kafka, etc.*
-If you're not outputting data directly from APM Server to Elasticsearch (perhaps you're using Logstash or Kafka),
-then the index template will not be set up automatically. Instead, you'll need to
-{apm-guide-ref}/apm-server-template.html[load the template manually].
-
-*Using a custom index names*
-This problem can also occur if you've customized the index name that you write APM data to.
-If you change the default, you must also configure the `setup.template.name` and `setup.template.pattern` options.
-See {apm-guide-ref}/configuration-template.html[Load the Elasticsearch index template].
-If the Elasticsearch index template has already been successfully loaded to the index,
-you can customize the indices that the APM app uses to display data.
-Navigate to *APM* > *Settings* > *Indices*, and change all `xpack.apm.indices.*` values to
-include the new index pattern. For example: `customIndexName-*`.
```

Comment:
Is it possible to write to custom data streams in 8.0?

Reply:
The data stream type and dataset cannot be customised, but the namespace can (set in the Fleet policy).

Reply:
Thanks, Andrew. In that case it sounds like this is safe to remove as the custom namespace won't break the APM app.